build.sbt

```diff
@@ -27,8 +27,7 @@ scalacOptions ++= Seq("-release", "8", "-target:jvm-1.8")
 core / libraryDependencies ++= Seq("org.rocksdb" % "rocksdbjni" % "6.6.4",
   "commons-codec" % "commons-codec" % "1.15",
   "org.apache.spark" %% "spark-sql" % "3.0.1" % "provided",
-  "com.thesamet.scalapb" %% "sparksql-scalapb" % "0.11.0-RC1",
-  "com.thesamet.scalapb" %% "scalapb-runtime" % "0.10.4" % "protobuf",
+  "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion % "protobuf",
```

Comment:
Do we still need this?  Makes me sad (and may make compiling for both Spark 2.4.7 and Spark 3.0.1 somewhat harder, I think they may require different versions).

Reply:
Unfortunately yes, this is what brings the protobuf types (e.g. google/protobuf/timestamp.proto).